ide-setup.md

```diff
@@ -51,7 +51,19 @@ To install the plugins:
    directory, without the `bin` subdirectory. For example, `c:\Users\obiwan\flutter` / `/Users/obiwan/flutter`.
 1. Click **OK**
 
-## Using the IDEs for Flutter
+### Using the IDEs for Flutter
 
 See our [additional documentation](/using-ide/) for tips on developing Flutter apps with
 Android Studio or IntelliJ IDEA.
+
+## Community supported IDEs
+
+There are number of additional IDE & editor options available enabled by community contributions:
+
+* [VS Code](https://code.visualstudio.com/) with the [Dart Code plugin](https://marketplace.visualstudio.com/items?itemName=DanTup.dart-code)
```

Comment:
Perhaps:

`with the [Dart Code](https://marketplace.visualstudio.com/items?itemName=DanTup.dart-code) plugin`

Reply:
"plugin for VS Code" (for SEO)